features/product/viewing_products/seeing_correct_lowest_prices_when_period_changes_on_the_channel.feature

```diff
@@ -0,0 +1,52 @@
+@viewing_products
+Feature: Seeing the correct product's lowest price when the period on the channel changes
+    In order to know whether the current discount is attractive
+    As a Guest
+    I want to see a product's lowest price within the channel's set period
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And it is "2023-01-01" now
+        And the store has a product "Wyborowa Vodka" priced at "$10.00"
+        And on "2023-01-05" its price changed to "$11.00"
+        And on "2023-01-10" its price changed to "$12.00" and original price to "$20.00"
+        And on "2023-01-15" its price changed to "$13.00" and original price to "$15.00"
+        And on "2023-01-20" its price changed to "$14.00" and original price to "$10.00"
+        And on "2023-01-25" its price changed to "$15.00" and original price to "$5.00"
+        And it is "2023-01-30" now
```

Comment:
```suggestion
        And it is "2023-01-31" now
```
without this change, in my opinion, the below scenarios are not 100% correct, because for example 25 days ago (scenario line 25) there was 2023-01-05 and on that day the price changed, so the previous price ($10) was still valid, so there should be $10 instead of $11 🤔

Reply:
I get your point, but this particular date doesn't change anything, because we calculate based on the last promotion date, not now. Frankly, this step can be removed.